src/replication_misc.c

```diff
@@ -310,3 +312,19 @@ set_socket_keepalive(int sfd)
 out:
 	return (ret);
 }
+
+json_object *
+json_object_new_uint64(uint64_t value)
+{
+	/* 18446744073709551615 */
+	char num[21];
```

Comment:
ok so we are sending as string, but is it certain that value will never go beyond 21 chars ?